baseplate/observers/sentry.py

```diff
@@ -89,6 +106,25 @@ def error_reporter_from_config(raw_config: config.RawConfig, module_name: str) -
         else:
             break
 
+    cfg_ignore_exceptions = cfg.sentry.ignore_exceptions
+    cfg_additional_ignore_exceptions = cfg.sentry.additional_ignore_exceptions
+
+    if cfg_ignore_exceptions != None:
```

Comment:
```suggestion
    if cfg_ignore_exceptions:
```

since it defaults to `[]` would this have ever been `None`? if we do expect `None` it'd be more pythonic to write `is not None`